cli/pack/app_dir.go

```diff
@@ -110,9 +112,9 @@ func initAppDir(appDirPath string, ctx *context.Ctx) error {
 	return nil
 }
 
-func copyRocksFromCache(cachedRocksDir string, appDirPath string) error {
-	log.Info("Adding cached rocks")
-	err := copy.Copy(cachedRocksDir, filepath.Join(appDirPath, ".rocks"))
+func copyFromCache(cachedRocksDir string, appDirPath string) error {
+	log.Infof("Adding cached module %s", filepath.Base(appDirPath))
```

Comment:
`appDirPath` - `appDir` prefix confuses, app-dir is the directory in the resulting package. 
I suggest renaming:
* `copyFromCache` -> `copyPathFromCache`
* `cachedRocksDir` -> `cachedPath`
* `appDirPath` -> `destPath`

`"Adding cached module %s` -> `"Using cached path %s`